### PR TITLE
Avalonia 11 upgrade

### PR DESCRIPTION
**Pull request by joaompneves**

### CODE REVIEW

**Comment by sandreas** [comment]
@joaompneves I really appreciate, that you took a look at this, thank you.

I hope you don't mind me asking a question, because I don't understand what you are doing [here](https://github.com/OutSystems/CefGlue/blob/6adaf7f1ebbbf8048d7ca7737b8200afff6eaf89/CefGlue.Avalonia/AvaloniaBrowserProcessHandler.cs#L28):

```c#
_current = Observable.Interval(TimeSpan.FromMilliseconds(delayMs)).ObserveOn(AvaloniaScheduler.Instance).Subscribe((i) => {
    CefRuntime.DoMessageLoopWork();
});
```

It feels a little weird to me adding the whole `Avalonia.ReactiveUI` Dependency only for supporting this small piece of code. I personally switched from `Avalonia.ReactiveUI` to `CommunityToolkit.Mvvm` and while it would not be a real problem to use `ReactiveUI`, maybe this can be solved without being a mandatory dependency?

**Comment by joaompneves** [comment]
> @joaompneves I really appreciate, that you took a look at this, thank you.
> 
> I hope you don't mind me asking a question, because I don't understand what you are doing [here](https://github.com/OutSystems/CefGlue/blob/6adaf7f1ebbbf8048d7ca7737b8200afff6eaf89/CefGlue.Avalonia/AvaloniaBrowserProcessHandler.cs#L28):
> 
> ```cs
> _current = Observable.Interval(TimeSpan.FromMilliseconds(delayMs)).ObserveOn(AvaloniaScheduler.Instance).Subscribe((i) => {
>     CefRuntime.DoMessageLoopWork();
> });
> ```
> 
> It feels a little weird to me adding the whole `Avalonia.ReactiveUI` Dependency only for supporting this small piece of code. I personally switched from `Avalonia.ReactiveUI` to `CommunityToolkit.Mvvm` and while it would not be a real problem to use `ReactiveUI`, maybe this can be solved without being a mandatory dependency?

I'm just trying to make it work, and the AvaloniaScheduler moved to that package, that's why I imported it. Most important this is still not working.

**Comment by joaompneves** [comment]
It's now displaying. Regarding your previous comment, [Avalonia.ReactiveUI](https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.ReactiveUI) is a very small dependency, and essential for CefGlue to work on mac. `CommunityToolkit.Mvvm is` not the same thing.

**Comment by joaompneves** [comment]
The [package](https://www.nuget.org/packages/CefGlue.Avalonia/106.5249.7-avalonia11) is now available.

File: CefGlue.Avalonia/InputExtensions.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 using Avalonia.Input;
 using Avalonia.VisualTree;
 using System.IO;
+using Avalonia;
```
Comment (alvesmiguel1) [inline CefGlue.Avalonia/InputExtensions.cs:4]: usings are not sorted

**Review by alvesmiguel1 — DISMISSED** [review]

**Comment by ITDancer13** [comment]
Is there anything I could do to support this PR? Would be great to get it merged :-)

File: CefGlue.Avalonia/InputExtensions.cs (hunk 1 above)
Comment (JMano) [reply CefGlue.Avalonia/InputExtensions.cs:2]: They should be now 👍🏽

**Review by alvesmiguel1 — DISMISSED** [review]

**Review by alvesmiguel1 — DISMISSED** [review]

**Review by alvesmiguel1 — APPROVED** [review]

File: Directory.Packages.props [hunk 2]
```diff
@@ -14,13 +13,15 @@
...
-        <PackageVersion Include="Avalonia.Diagnostics" Version="0.10.17" />
-
+        <PackageVersion Include="Avalonia" Version="11.0.9" />
+        <PackageVersion Include="Avalonia.Desktop" Version="11.0.9" />
+        <PackageVersion Include="Avalonia.Diagnostics" Version="11.0.9" />
+        <PackageVersion Include="Avalonia.ReactiveUI" Version="11.0.9" />
+        <PackageVersion Include="Avalonia.Themes.Simple" Version="11.0.9" />
+      
         <PackageVersion Include="Dotnet.Bundle" Version="0.9.13" />
+        <PackageVersion Include="System.Reactive.Linq" Version="6.0.0-preview.9" />
```
Comment (joaompneves) [inline Directory.Packages.props:24]: theres a new version

**Pull request merged**